config/homebin/db_backup

```diff
@@ -53,15 +53,15 @@ ext=".sql"
 if [[ "${gzip}" == "True" ]]; then
     ext=".sql.gz"
 fi
-count=0
+count=1
```

Comment:
Would this mean the count is always 1 or higher even if nothing got backed up?

Reply:
count starts at 1 if there's something, variable is not used at all if nothing goes through the loop